packages/main/src/Tokenizer.js

```diff
@@ -225,7 +229,16 @@ class Tokenizer extends UI5Element {
 	}
 
 	onAfterRendering() {
-		this._scrollEnablement.scrollContainer = this.expanded ? this.contentDom : this;
+		this._scrollEnablement.scrollContainer = (this.expanded || !this.narrowContentDom) ? this.expandedContentDom : this.narrowContentDom;
+
+		if (this.expanded) {
```

Comment:
why not if/else?

Reply:
To avoid another if/else block for less code.